creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -84,13 +84,33 @@ async function _primarySyncFromSecondary({
     })
```

Comment:
similar change to lines 55 and 69 above?

Reply:
will add to lines 55 and 69